Preparation/TestProject1/MyStackInterTest.cs

```diff
@@ -0,0 +1,59 @@
+using MyStack;
+using Xunit;
+
+namespace Preperation.Tests
+{
+    public class MyStackInterTest
+    {
+        [Fact]
+        public void PushTest()
+        {
+            MyStack<int> myStack = new MyStack<int>();
+
+            myStack.Push(1);
+
+            Assert.Equal(1, myStack.Peek());
+        }
+        [Fact]
+        public void PeekTest()
```

Comment:
I would test that Peek takes the last pushed element, but doesn't change the count of the elements.